internal/app/machined/pkg/controllers/k8s/kubelet_service.go

```diff
@@ -196,6 +196,25 @@ func (ctrl *KubeletServiceController) Run(ctx context.Context, r controller.Runt
 			}
 		}
 
+		oldPolicy, err := loadPolicyFromFile("/var/lib/kubelet/cpu_manager_state")
```

Comment:
I think this new code should be moved out to function which manages policy change like other functions here

Reply:
Done